ingestors/kafka/src/main/scala/hydra/kafka/algebras/KafkaClientAlgebra.scala

```diff
@@ -304,24 +326,42 @@ object KafkaClientAlgebra {
         consumerStream(consumerSettings)
           .evalTap(_.subscribeTo(topicName))
           .flatMap(_.stream)
-          .evalTap{ committable =>
+          // TODO: Commit before action? do we need at least once delivery or at most once?
```

Comment:
We need at least once delivery. We have an eventual consistency model.

Reply:
Then, we will need to fix this part